extensions/database/module/styles/database-import.css

```
@@ -161,7 +161,7 @@
 .sc-context-more-vert:after {
```

```suggestion
.sc-context-more:after {
```

I'm not sure changing this is worth it, but it caught my eye because of the icon name change. If it gets changed here, it would also mean changing the four references in `database-source-ui.js` and `database-extension.js` (which probably means it's not worth the hassle, but I'll leave it up to you.)